packages/changed-elements-react/README.md

```diff
@@ -33,14 +35,69 @@ To begin using this package in your application, you will need to:
     });
     ```
 
-4. Mount `<VersionCompareSelectComponent />` to begin version comparison worflow by selecting iModel versions to compare. You should also add a button in your UI that calls `VersionCompare.manager.stopComparison()`.
-5. `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility.
+4. `<ChangedElementsWidget />` React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.
+
+    ```ts
+    import { ChangedElementsWidget } from "@itwin/changed-elements-react";
+    <ChangedElementsWidget
+          useV2Widget
+          feedbackUrl="https://example.com"
+          iModelConnection={UiFramework.getIModelConnection()!}
+          enableComparisonJobUpdateToasts
+          manageNamedVersionsSlot={<ManageNamedVersions />}
+          documentationHref="https://example.com"
+        />,
+    ```
+
+5. `<NamedVersionSelectorWidget />` **Experimental** React component lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code has example of widget use but does not show off all props for widget.
```

Comment:
Here is a suggested change, but also, should this say the same thing as the bullet point above? It's doing the same stuff?
```suggestion
5. The `<NamedVersionSelectorWidget />` is an **experimental** React component that lets users inspect differences in properties between versions, generate reports, search for changed elements, and control element visibility. The following code shows an example widget initialization (does not show off all props).
```